sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/RosterBean.java

```diff
@@ -338,17 +339,16 @@ public void export(ActionEvent event){
         	
         } 
         spreadsheetData.add(header);
-        for(Iterator enrollmentIter = siteStudents.iterator(); enrollmentIter.hasNext();) {
-            //EnrollmentDecorator enrollment = enrollmentIter.next();
-        	ParticipationRecord record = (ParticipationRecord)enrollmentIter.next();
+        for(Iterator enrollmentIter = unpagedEnrollments.iterator(); enrollmentIter.hasNext();) {
```

Comment:
Do you need to use an iterator here? 

`for (EnrollmentDecorator enrollment  : unpagedEnrollments ) {`